build_runner/lib/src/generate/fold_frames.dart

```diff
@@ -8,6 +8,7 @@ const _foldFramesFor = const [
   '_bazel_codegen',
   'build',
   'build_barback',
```

Comment:
Remove this?

Reply:
if it showed up (you can still pass the `BarbackResolvers` through if you want) we'd want it folded. We should remove this if and when we delete the package.